lib/emoji.js

```diff
@@ -126,3 +127,30 @@ Emoji.search = function search(str) {
     };
   });
 }
+
+Object.prototype.getKeyByValue = function(value) {
+  for(var prop in this) {
```

Comment:
please stick to the style in the rest of the library (space after `for` and `if`)